### DISCUSSION TITLE
Location of cmd prompt

### DISCUSSION THREAD

**Discussion by limaceous-bushwhacker** [jarun/nnn#1730] [Q&A]
Was the location changed recently? It used to show up just above the statusline before but now it opens a new shell(?) just like `!` does.

**Comment by limaceous-bushwhacker** [comment]
Looks like I can get back the old placement using `-N` flag, but I'm still curious why this change was made or is there a reason that it's now the default?

**Reply by jarun** [reply]
I don't understand. What was the original set of options you were using? Is you binary compiled with libreadline? Please share those details.

If readline is linked, without `-N` it should open like a new shell. With `-N` it should show up at the status line.
No change in behaviour is expected without libreadline linkage.

**Reply by limaceous-bushwhacker** [reply]
I wasn't using any flag in particular. Was "native prompt" always supposed to open a new window? Because I remember it just opening above the status line. What's the difference between native prompt and shell prompt? Might as well open up a shell prompt instead of the plain native prompt.
